compass/landice/mesh.py

```diff
@@ -60,3 +61,43 @@ def gridded_flood_fill(field):
         lastSearchList = newSearchList
 
     return floodMask
+
+
+def set_geom_points_and_edges(xx0, xx1, yy0, yy1):
+    """
+    Set node and edge coordinates to pass to build_planar_mesh().
```

Comment:
```suggestion
    Set node and edge coordinates to pass to 
    :py:func:`mpas_tools.mesh.creation.build_mesh.build_planar_mesh()`.
```
If you do this, it should automatically link to that function in the MPAS-Tools API, which can add clarity and make it easier to navigate.

Reply:
Oh great, thanks!